2019-05-Berlin/agenda.md

```diff
@@ -0,0 +1,95 @@
+|---------------|--------------|-----------------------------------------------|----------------------------------------|--------------------------------------------|-------------------------------------------|
+| | Main         | Intro, Sponsor Talk, Code of Conduct          | |                                            | |
+
+
+_To speak/present a topic/do a sesssion, make a pull request to update this document with a title, abstract, and a link to an issue with more detail in this repo, similar to https://github.com/nodejs/summit/issues/86_.
+
+
+## Collaborator Summit Day Two: 31st May, 2019
+
+(time window)
+
+### Agenda
+
+(Types of rooms available):
+
+* (details of main room)
+* (details of more rooms)
+
+_(Expected participation number)_.
+
+
+| Time          | Place        | Topic                                                               | Speakers/Facilitators            | Issue                                      | Slides                                    |
+|---------------|--------------|---------------------------------------------------------------------|-------------------------------|--------------------------------------------|-------------------------------------------|
+| | Main | Intro, Code of Conduct                                              | |                                            | |
+
+
+_To speak/present a topic/do a sesssion, make a pull request to update this document with a topic, contact @github-handle. Include a link to an issue in this repo with more detail, similar to https://github.com/nodejs/summit/issues/86_ that contains at least an abstract, a link to the planning issue in the working group repo, and any extra requests (minimum space during conf, av/sound, etc.).
+
+
+## Organizers
+
+- Matteo Collina (@mcollina)
+- Manil Chowdhury (@chowdhurian)
+-  (@trivikr)
+-  (@thefourtheye)
+-  (@mmarchini)
+-  (@sagirk)
+-  (@WaleedAshraf)
+-  (@joyeecheung)
+-  (@jorydotcom)
+
+## Attendance
+
+Anyone can come, but we will not be explicitly onboarding at Collab 
+Summit Vancouver. Conversations can move fast as working groups have 
+a lot of context. Unlike the Fall instance of the Collab Summit, this Spring instance does not include a morning of Code + Learn, the session that guides people through contributing to the project.
+
+Working groups will put together a brief schedule so that people can 
+familiarize themselves before folks get onsite, having the general collaborator 
+discussions, and then dive into breakout sessions.
+
+Attendees must register at: (TBA)
+
+  
+## Travel Fund
+There are funds available for any Individual Member of the Node.js Foundation to receive travel funds to Collab Summit.  Membership is free for active collaborators. Sign up [HERE](https://github.com/nodejs/members#members).
```

Comment:
@chowdhurian Links seems to 404 here

Reply:
@mbj36 This is a private repo, so 404 is expected.

Reply:
Thanks for clarifying